contracts/controller/UController.sol

```diff
@@ -280,6 +280,23 @@ contract UController is ControllerInterface {
         return organizations[_avatar].schemes[_scheme].permissions;
     }
 
+    function getGlobalConstraintParameters(address _globalConstraint, address _avatar) external view returns(bytes32) {
```

Comment:
please return 2 value : pre and post params

Reply:
As I read the code, both pre and post will have exactly the same params. Is that not the case?